uint8[10]          robot_id
motor_speed[10]    robots_speed